tests/Couchbase.UnitTests/Fixtures/ClusterFixture.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using System.Threading.Tasks;
+
+namespace Couchbase.UnitTests.Fixtures
+{
+    public class ClusterFixture : IDisposable
+    {
+        public ICluster Cluster { get; }
+
+        public ClusterFixture()
+        {
+            var cluster = new Cluster();
+            var task = cluster.Initialize(
+                new Configuration()
+                    .WithServers("couchbase://10.112.193.101")
```

Comment:
This should probably be "couchbase://127.0.0.1"  by default and then modified when you run locally.